### DISCUSSION TITLE
APY3000%，年化30倍是草根的梦想，还是幻想？

### DISCUSSION THREAD

**Discussion by yancanlou** [famland/website#92] [Show and tell]
在流行梦想照进现实的当下，谈梦想也不是一种奢望了，成了一种必须品，而似乎没有梦想才是最可怕的。
本着梦想可以实现的设想，那么我们不防扪心自问下，所谓的年化3000%是梦想还是幻想？
**常言道，人生只需要7次倍投。**
2^7=128，当你的资产实现了这个128倍，就足以领先你周边90%以上的人率先实现梦想了。
**如果这个梦想能通过每年翻倍，理想情况下不要能说连续7年达成，即使是10年以上奋斗达标，也已经是人生赢家。**
然而在币圈，却总有人一而再三地描述了一夜百倍的暴富神话，10年128倍的梦想反而成了一只心无斗志的丑小鸭，再无人多看一眼。
在资本游戏者地鼓吹下，这其实是泡沫放大梦想以后的悲哀，也必终将迎来梦想的泡沫化，哪道失败的例子还不够吗？
然而草根的世界里，唯一不缺的就是堵徒心态，总想给自己一个赌赢的机会，这就注定了资金盘的此起彼伏，一边不断破灭，一边不断再生~反反复复。
在这样一个时代背景下，草根能不能有一个通过持续量变达到质变的机会呢？
稳定币的兴起，似乎给出了一个这样实现财富的机会，但同样也结出了更多的资金盘。
在稳定币的浪潮中，如何能分辨出是资金盘还是梦想盘？
**个人以为一个最重要的指标，就是其生态能否可持续性发展，说白了，就是它能经得起时间的考验吗？
这个经得起考验才是核心指标。**
转换一种表达，其它都不要说，回到题目，3000%年化是不是一个可持续发展的设想？
这样一个口号或者这样一种游戏在个人看来是不可实现的。
**就拿比特币来说，达到2000万倍这种神话，也是历经10年成长，其平均年化也只是## 530%##而已，这已经是神话。**
当提出3000%年化这个口号时，这个本身就被当成是资金盘来围观了。
**一个完全去中心化的项目，应当有一个长远的可以实现地梦想，而不是让这种梦想被自己的口号所束缚，如何群策群力下，实现可持续发展才是社区永久要面临的问题，这是才是存亡之道。**
**建议：口号真不要打得太过了，会吓走正常的草根与理性的资本，只会招来堵徒或培养成堵徒。**
年化3000%要实现也要极端条件才能有，只能是那种万中无一。
**一种是持续性发展多年后，少数的头部全超高共享值合田后一伙顶级玩家农场内才有的存在。、
一种就是短期前几个创世周期，刺激性的短效存在，然后永久关掉。**
**不然无论何种地成长模型是支持不了这个年化口号的。**

所谓欲速则不达，最高年化500%，能发展下去就足以远超比特币传说！
**说现实点，哪怕只有其区区的十分之一，平均年化50%，也就是12年129倍，那么能维持生态两年也可以傲视全球了。**
共勉！希望社区能靠成员共建共享实现梦想，而不是让其成为幻想！

**Comment by cryptosoul-vioiv** [comment]
3000%是可以达到的，同时也是封顶值；但可以维持多久，就是要共享农场如何来打造。

**Reply by yancanlou** [reply]
一时？
没理解这个口号的伤害性啊
从没见过几秒内赚了几块钱就说自己年化是3000%，这样每个项目都是瞬时年化3000%以上。
提这个说法会产生不好影响，少了一个会务实社区氛围，要提就得平均年化。

**Reply by otto-001** [reply]
3000%只是白皮书里的封顶值，不是口号😁😂。
通过加强共识，可以达到的一个顶

**Reply by cryptosoul-vioiv** [reply]
赞同。以务实的平均年化，是最妥的。
其实，哪怕平均年化能在300%平稳运行，已经足够秒杀99%的项目了。
